bundles/org.eclipse.ui.ide/extensions/org/eclipse/ui/actions/WorkspaceModifyOperation.java

```diff
@@ -103,20 +103,17 @@ protected abstract void execute(IProgressMonitor monitor)
 	@Override
 	public synchronized final void run(IProgressMonitor monitor)
 			throws InvocationTargetException, InterruptedException {
-		final InvocationTargetException[] iteHolder = new InvocationTargetException[1];
+		final Exception[] exceptionHolder = new Exception[1];
 		try {
 			IWorkspaceRunnable workspaceRunnable = pm -> {
 				try {
 					execute(pm);
-				} catch (InvocationTargetException e1) {
+				} catch (InvocationTargetException | InterruptedException | CoreException e1) {
```

Comment:
Why is `CoreException` caught now? In fact, the changed code completely swallows a `CoreException`. An  IWorkspaceRunnable` is allowed to throw a `CoreException` and the containing code already properly handles that.

Reply:
Good point, this is unnecessary. 

Done in 12615e9efe647076b42e23ad1130bd965296e783